share/picongpu/tests/Synchrotron_plugin/lib/validate.py

```diff
@@ -84,7 +84,15 @@ def main(dataPath):
     e_p_y *= e_p_y_SI / e_w
     Energy_e = momentum_to_energy(e_p_y, const.m_e).max()  # convert momentum to energy
 
-    hist_data /= Energy_e  # go to delta space
+    hist_data /= Energy_e  # go to delta space.
+    # The momentum precision is 6 digits so the energy is 3 digits precision
+    if np.max(hist_data) > 1 + 1e-3:
+        raise SystemExit(
+            f"\nMax normalized energy of photons is {np.max(hist_data):.6f} but it should be less than 1 + 1e-3. \n"
+            "Is electron recoil on?\n"
+        )
```

Comment:
Do we still need this check after figuring out that full energy transfer is not possible? 
If not, let's go back to 1 (even if this is too high).